File: day01/r/part2.R
library(dplyr)

input = read.delim("input.txt", header = FALSE) %>% pull()

for (i in input) {
  for (j in input) {
    for (k in input) {
      if (i + j + k == 2020) {
        print(i)
        print(j)
        print(k)
        print(paste("Final Answer:", i * j * k))
        break
      }
    }
  }
}

File: day01/jl/part2.jl
ret = map(str -> parse(Int, str), readlines("input.txt"))

#print(ret)

for i in ret, j in ret, k in ret
	#@show (i+j)
	if i+j+k == 2020.
		println("Final answer: $(i*j*k)")
		break
	end
end